otwcode/otwarchive
# AO3-5865 Reorganise 'Edit multiple works' form

# Pull Request Checklist

* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)? (updated)
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5865

## Purpose

Reorganizes the 'Edit multiple works' form into correctly titled sections to better align with regular Edit and Post work forms. 

Adds the 'verbose' class to the form (so section labels show). Updates capitalization of relevant labels within related sub-forms etc to sentence case where required to match recommendation from [AO3-5865](https://otwarchive.atlassian.net/browse/AO3-5865) but committed separately and can be rolled back if not desirable.

## Testing Instructions

Review 'Edit multiple works' form. See: https://otwarchive.atlassian.net/browse/AO3-5865

## References

N/A

## Credit

Saphron (denerose) - she/her.


[AO3-5865]: https://otwarchive.atlassian.net/browse/AO3-5865?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
